src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.data.person;
+
+/**
+ * General Class for contact details of Person( Email, Address, Phone )

```

Comment:
No need to mention the child classes in the header comment. By doing so, you need to keep updating this comment later on.